src/nvim/option.c

```diff
@@ -7137,7 +7137,7 @@ int tabstop_at(colnr_T col, long ts, long *vts)
   int t;
   int tab_size = 0;
 
-  if (vts == 0 || vts[0] == 0) {
+  if (vts == NULL || vts[0] == 0) {
```

Comment:
This check was not changed in the patch, only the other two were. I changed that one too for consistency, should/shouldn't I do that?

Reply:
Sure. Check `NULL`, not 0, for pointers.